.github/ISSUE_TEMPLATE/2_bug.yml

```diff
@@ -0,0 +1,133 @@
+name: Report a bug
+description: |
+  You found a bug in Flutter causing your application to crash or
+  throw an exception, a widget is buggy, or something looks wrong.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for using Flutter!
+
+        If you are looking for support, please check out our documentation
+        or consider asking a question on Stack Overflow:
+
+          - https://flutter.dev/
+          - https://api.flutter.dev/
+          - https://stackoverflow.com/questions/tagged/flutter?sort=frequent
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
```

Comment:
Is `description` required? If not, I would vote for removing this line since it seems redundant with the label and the first checkbox.

Reply:
Removed the description